ecosystem/sep-0006.md

```diff
@@ -1034,12 +1034,12 @@ Name | Type | Description
 `status` | string | Processing status of deposit/withdrawal.
 `status_eta` | number | (optional) Estimated number of seconds until a status change is expected.
 `more_info_url` | string | (optional) A URL the user can visit if they want more information about their account / status.
-`amount_in` | string | (optional) Amount received by anchor at start of transaction as a string with up to 7 decimals. Excludes any fees charged before the anchor received the funds.
+`amount_in` | string | (optional) Amount received by anchor at start of transaction as a string with up to 7 decimals. Excludes any fees charged before the anchor received the funds. Should be equals to the quotes's `sell_asset` if a `quote_id` was used.
 `amount_in_asset` | string | (optional) The asset received or to be received by the Anchor. Must be present if the deposit/withdraw was made using quotes. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
-`amount_out` | string | (optional) Amount sent by anchor to user at end of transaction as a string with up to 7 decimals. Excludes amount converted to XLM to fund account and any external fees.
+`amount_out` | string | (optional) Amount sent by anchor to user at end of transaction as a string with up to 7 decimals. Excludes amount converted to XLM to fund account and any external fees. Should be equals to the quotes's `buy_asset` if a `quote_id` was used.
 `amount_out_asset` | string | (optional) The asset delivered or to be delivered to the user. Must be present if the deposit/withdraw was made using quotes. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
-`amount_fee` | string | (optional) Amount of fee charged by anchor.
-`amount_fee_asset` | string | (optional) The asset in which fees are calculated in. Must be present if the deposit/withdraw was made using quotes. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
+`amount_fee` | string | (optional) Amount of fee charged by anchor. If a `quote_id` was used, fees and fee assets should be detailed in the quote's `price_details` field.
+`amount_fee_asset` | string | (optional) The asset in which fees are calculated in. Must be present if the deposit/withdraw was made using quotes. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format). If a `quote_id` was used, fees and fee assets should be detailed in the quote's `price_details` field.
```

Comment:
If a quote_id was used, `amount_fee_asset` field should be left empty and detailed in the quote's price_details field.

Reply:
https://github.com/stellar/stellar-protocol/pull/1199/commits/5b12d88086c6a0e6579c442a34b752e77ef4e222